### PR TITLE
Move coverage view to management command

### LINKED ISSUE #187: Move coverage view to management command
The [coverage view](https://github.com/DjangoAdminHackers/django-linkcheck/blob/800beea5f77c5db62cb55983dca9edd2b49b972e/linkcheck/views.py#L22-L34) tests which models are covered by the linkcheck config and which aren't.
In my opinion, there is no need for a separate view in the CMS for this. Only server admins who install the library should care about the configuration, not the staff users who are using the link checker in their workflow.

So I suggest to move the coverage view to a management command instead.

### PR DESCRIPTION
**Pull request by timobrembeck**
* Add new management command `linkcheck_suggest_config`
  * Deprecate coverage view

Fixes #187

### CODE REVIEW

**Review by claudep — APPROVED** [review]
I'm fine with that. I must admit I never used that `coverage`view!

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,8 @@
 Unreleased
 
+* Move coverage view to management command (Timo Ludwig, #187)
+  * Add new management command `linkcheck_suggest_config`
+  * Deprecate coverage view
```
Comment (claudep) [inline CHANGELOG]: Shouldn't this read "Delete coverage view"?

Comment (claudep) [reply CHANGELOG]: Hey Timo, what about answering this and pushing this pull request forward?

Comment (timobrembeck) [reply CHANGELOG]: Hey, sorry I was quite occupied with other projects (and still am), I will revisit this in November :wink:

Comment (claudep) [reply CHANGELOG]: OK, fine. No stress!

**Pull request merged**